battleship.py

```diff
@@ -255,7 +255,12 @@ def isHorizontal(ship):
 Returns: list of ints
 '''
 def getClickedCell(data, event):
-    return
+    l=[]

```

Comment:
Maintain a proper naming conventions for variables

Reply:
done